resources/base.css

```diff
@@ -65,6 +65,14 @@ h6 {
   margin-right: 1em;
 }
 
+.mb-1_2 {
+  margin-bottom: 0.5em;
+}
+
+.mb-1 {
+  margin-bottom: 1em;
+}
```

Comment:
Can you rename these classes to be less esoteric?

Reply:
Those are pretty commonly used generic class names in a lot of CSS templates for margin and padding. Seems esoteric at first but you get used to it so quickly that you don't even have to think about it imho ("m" or "p" for margin and padding, then optional direction, then em spacing... "mr-1_2" or "p-2" is way more convenient than writing "margin-bottom-1_2em" / "padding-all-2em" as a class name, since you need them quite often as well).

I can maybe add a comment block for explanation in the CSS file?

Reply:
If you think the names are okay, that's fine, just wanted to make sure this wasn't some sort of oversight because the names seemed a little esoteric at first :-)

I don't think an explanatory comment is necessary- it's obvious what the classes do when you read them.